README.md

```diff
@@ -112,12 +112,12 @@ engine = create_engine(
 )
 ```
 
-## Authentications
+## Authentication Mechanisms
```

Comment:
This should be "Authentication mechanisms" (lowercase `M`) if we want to make it compatible with Trino documentation. This is the suggested rule in Google technical writing if I remember correctly. 

(Follow-up PR is also fine)

Reply:
Thank you. I'll fixup and squash.